events.go

```diff
@@ -353,3 +353,58 @@ func (eventTracerDisabled) EventTracerDisabled() {}
 func (eventTracerDisabled) String() string {
 	return tracerDisabled
 }
+
+type EventSystemMetricsMeasurementFailed interface {
+	ErrorEvent
+}
+
+type eventSystemMetricsMeasurementFailed struct {
+	err error
+}
+
+func newEventSystemMetricsMeasurementFailed(err error) *eventSystemMetricsMeasurementFailed {
+	return &eventSystemMetricsMeasurementFailed{
+		err: err,
+	}
+}
+
+func (e *eventSystemMetricsMeasurementFailed) Event() {}
+
+func (e *eventSystemMetricsMeasurementFailed) String() string {
+	return e.err.Error()
```

Comment:
nit: Either `#String()` or `#Error()` should probably piggyback off the other

Reply:
Fixed. Updated the rest of the events as well